frontend/static_vol/js/modules/modal.js

```diff
@@ -68,15 +72,57 @@ const closeModalOnCancel = () => {
                 indicatorBar.removeEventListener('transitionend', endIndicator);
             }
             //cancel game
-            cancel_game();
+            console.log(`cancel game: ${username}`);
+            cancel_game(username);
         })
         .then(() => {
             //modal close
             closeModal();
         });
 }
 
-const closeModalOnAccept = () => {
+const closeModalOnReject = (ev) => {
+    console.log('closeModalOnReject');
+    const modal = ev.target.closest('.blockModal');
```

Comment:
```suggestion
const closeModalOnReject = () => {
    console.log('closeModalOnReject');
    const modal = document.querySelector('.blockModal');
```

相手に断られた場合こちらではevが発生せずundefinedになってしまうので
querySelectorが良さそうです。
blockModal はページ内に1つしかないはずですし

Reply:
closeOnRejectはリジェクトする側の処理ですのでev渡ってきます。
ログでおかしいのはconst reject_gameの書式がミスってる（バッククオートになってない）せいですね。
 ```javascript
    console.log('accept_game ${request_id}')
 ```
こっち直しておきます。

Reply:
なるほど確かに！
 失礼しました🙇